src/Shell/Task/ModelTask.php

```diff
@@ -875,6 +875,16 @@ public function listUnskipped()
     }
 
     /**
+     * Models never have routing prefixes applied.
+     *
+     * @return string
+     */
+    protected function _getPrefix()
```

Comment:
Renaming the method makes sense, changing the CLI flags around less so, as each prefix is contextual.